10up/ElasticPress
# Hide taxonomies from facet block whose show_ui is set to false.

### Description of the Change

This PR hides the taxonomies from facet block whose `show_ui` is set to false.


Closes #2957

### How to test the Change


- Go to Widgets screens
- Add Facet Block 
- Make sure Taxonomy field does not have `Custom Result` option

### Changelog Entry

> Changed - Hide taxonomies from facet block whose show_ui is set to false.


### Credits

Props @burhandodhy 


### Checklist:
